kubejs/config/common.properties

```diff
@@ -1,7 +1,8 @@
 #KubeJS Common Properties
-#Sat May 01 23:14:14 CEST 2021
+#Sat Sept 28 21:45:14 CEST 2021
 exportVisualizerData=false
 packmode=default
 hideServerScriptErrors=false
 announceReload=true
 serverOnly=false
+invertClassLoader=true

```

Comment:
This setting change worries me to be honest 😄 
@TemperedFool did you create something that relied on this being false?

Reply:
I asked on the KubeJS Discord and am no longer worried 😄